api/src/shared/application/scripts/script-runner.js

```diff
@@ -0,0 +1,60 @@
+import * as url from 'node:url';
+
+import pick from 'lodash/pick.js';
+import { hideBin } from 'yargs/helpers';
+import yargs from 'yargs/yargs';
+
+import { disconnect } from '../../../../db/knex-database-connection.js';
+import { learningContentCache } from '../../infrastructure/caches/learning-content-cache.js';
+import { logger as defaultLogger } from '../../infrastructure/utils/logger.js';
+
+function isRunningFromCLI(scriptFileUrl) {
+  const modulePath = url.fileURLToPath(scriptFileUrl);
+  return process.argv[1] === modulePath;
+}
+
+/**
+ * A utility class for running scripts from the command line.
+ */
+export class ScriptRunner {
```

Comment:
Adapter le nom de la classe du Script Runner suivant la suggestion de modifier le nom de la classe de base pour les scripts : 
```suggestion
export class ManagedScriptRunner {
```